cmd/root.go

```diff
@@ -88,6 +90,15 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	if clowder.IsClowderEnabled() {
+		options.Storage.Postgres.Host = clowder.LoadedConfig.Database.Hostname
+		options.Storage.Postgres.Port = strconv.Itoa(clowder.LoadedConfig.Database.Port)
```

Comment:
Have you verified the order for loading the configuration? If Clowder is enabled and loading `inventory-api-config.yaml` will not over-ride the config loaded from the clowder?

Reply:
Adding this here from our conversation:
The the current inventory config is only setting `storage.database` and `storage.sqlite3.dsn`, as I was hesitant to hard code those into the new code. If i remove those settings from the inventory-api-config, the init container file fails with

```shell
INFO ts=2024-09-26T13:37:57Z caller=storage/storage.go:24 service.name=inventory-api service.version=0.1.0 trace.id= span.id= group=storage group=storage msg=Using backing storage: sqlite3

2024/09/26 13:37:57 /workspace/internal/storage/storage.go:25
[error] failed to initialize database, got error unable to open database file: no such file or directory
```

so it seems like config is loaded despite command flags being set, and since none of the values are getting overwritten, it's accepting all values.